src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -879,18 +977,27 @@ private CompletableFuture<MessageID> splitSend(IChannel channel, ByteBuf message
         slices.add(message.retainedSlice(readerIndex + nextSliceStart, message.writerIndex() - nextSliceStart - readerIndex));
 
         long sleepTime = config.getTimeoutBetweenParts();
-        List<CompletableFuture<MessageID>> messageIDS = new ArrayList<>();
+        List<Instant> sendingTimes = new ArrayList<>();
         for(ByteBuf slice : slices) {
-            messageIDS.add(channel.send(slice, properties, eventID, SendMode.DIRECT));
+            channel.send(asExpandable(slice), metadata, eventID, SendMode.DIRECT_SOCKET);
+            sendingTimes.add(Instant.now());
             try {
                 Thread.sleep(sleepTime);
             } catch (InterruptedException e) {
                 LOGGER.error("Error while sending messages in different tcp packets.");
             }
         }
-        return messageIDS.get(0);
+
+
+        String slicesTimestamps = sendingTimes.stream().map(formatter::format).reduce((i1, i2) -> i1 + "," + i2).orElse("");
```

Comment:
I think we can use existing collector here instead
```suggestion
        String slicesTimestamps = sendingTimes.stream().map(formatter::format).collect(Collectors.joining(","));
```